src/resources/js/components/EditPresentation.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+    <v-form
+        ref="form"
+        v-model="valid"
+        lazy-validation
+    >
+      <v-text-field
+            v-model="updatePresentationInfo.title"
+            :rules="titleRules"
+            label="Titre"
+      />
+
+      <v-text-field
+            v-model="updatePresentationInfo.date"
+            label="Date"
+      />
+
+      <v-btn
+            :disabled="!valid"
+            color="success"
+            class="mr-4"
+            @click="submitChange"
+            >
+            Appliquer
+      </v-btn>
+    </v-form>
+</template>
+
+<script>
+  let alert = {};
+  export default {
+    props: ['parentRefs'],
+    data() {
+      return {
+        valid: true,
+        titleRules: [
+          (v) => v === '' || (v && v.length <= 20) || 'Le titre doit contenir moins de 20 caractères',
+        ],
+        updatePresentationInfo: {
+          date: new Date(),
```

Comment:
what if we do not want to change the date ?

Reply:
Also make sure the format in your date object is `YYYY-MM-DD HH:mm` see #145 for more details

Reply:
Changed in 8a5590f